packages/terra-form-validation/docs/README.md

```diff
@@ -0,0 +1,196 @@
+# Form Validation
+
+To write form validations for your component, Terra recommends using [react-final-form](https://github.com/final-form/react-final-form). This package contains several different hooks and functionality for performing validations at various levels which include:
+
+- Validating onChange
+- Validating onSubmit
+- Validation synchronously (Such as for unit user name implementations)
+- Restricting inputs to a particular format.
+
+For other functionality that is provided, consult the package's [documentation](https://github.com/final-form/react-final-form).
+
+## Getting Started
+
+Import Form and Field from react-final-form, and build your form like the following, where Form wraps the entire form, and Field is used for each individual import on the form.
+
+```jsx
```

Comment:
I'd cut this section, we don't want people using react-final-form's `Field` component without terra's components as it will render a different UI than intended.